app/views/SidebarView/index.js

```diff
@@ -103,9 +105,16 @@ class Sidebar extends Component {
 		if (nextState.isAdmin !== isAdmin) {
 			return true;
 		}
+		if (nextProps.state !== state && isSetAdminNeedCall === true) {
+			return true;
+		}
 		return false;
 	}
 
+	componentDidUpdate() {
+		this.setIsAdmin();
```

Comment:
Not a good practice to call something on `componentDidUpdate` without testing in what cases it should run.
`componentDidUpdate` runs every time the component updates: when state changes, when props change or when the component is forced updated.

Even if `shouldComponentUpdate` returns false, `this.setIsAdmin` is being called.
Add a `console.log` at the start of `setIsAdmin` and you'll see how many times it's called.